src/azure-cli/azure/cli/command_modules/cognitiveservices/tests/latest/test_cognitiveservices_command.py

```diff
@@ -82,7 +82,15 @@ def test_cognitiveservices_account_list_skus_legacy(self, resource_group):
                  checks=[self.check('name', '{name}'),
                          self.check('location', '{location}'),
                          self.check('sku.name', '{sku}'),
-                         self.check('properties.provisioningState', 'Succeeded')])
+                         ])
+
+        for i in range(10):
+            account = self.cmd('az cognitiveservices account show -n {name} -g {rg}').get_output_in_json()
+            if 'Creating' != account['properties']['provisioningState']:
+                break
+            time.sleep(15)
```

Comment:
Could you use command `az resource wait --ids resource-id --created` to monitor the status of resource instead of `sleep()` logic?

Reply:
Yeah, this is a feasible way before. Now track2 has a better alternative. I've corrected them all